packages/big-design/src/components/Select/types.ts

```diff
@@ -38,3 +38,8 @@ export interface SelectOption<T> extends Omit<BaseItem, 'actionType'> {
 export interface SelectAction extends BaseItem {
   onActionClick(inputText: string | null): void;
 }
+
+export interface SelectOptionGroup {
+  optionsLabel: string;
+  options: Array<SelectOption<any>>;
+}
```

Comment:
I believe `SelectOptionGroup` should accept a `<T>` type that is passed down to `SelectOption` so that we don't lose the type of the value.

Reply:
Had some trouble passing `<T>` type to `SelectOptionGroup` which is why I passed `<any>`, assumed you **_could_** pass `<any>` type and then the `SelectOptionGroup` itself would apply the generic typing to it - happy to go through it and try something different though.